docs/reference/search/search-your-data/semantic-search-elser.asciidoc

```diff
@@ -97,25 +99,24 @@ PUT _ingest/pipeline/elser-v2-test
     {
       "inference": {
         "model_id": ".elser_model_2",
-        "target_field": "ml",
-        "field_map": { <1>
-          "text": "text_field"
-        },
-        "inference_config": {
-          "text_expansion": { <2>
-            "results_field": "tokens"
+        "input_output": [ <1>
+          {
+            "input_field": "content",
+            "output_field": "content_embedding"
           }
+        ] 
+      },
```

Comment:
This code snippet has the same issue that was fixed in https://github.com/elastic/elasticsearch/pull/100420/commits/3c63fd85cce65cff8713500ec653e8f7f80f0479 and https://github.com/elastic/elasticsearch/pull/100420/commits/b56fd256c96ff4daf528099d8946a85f965f9413 for the other snippet. It wasn't caught because this snippet is not tested. We should consider not skipping this test.